extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -414,6 +416,28 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     });
   }
 
+  /** @override */
+  getServeNpaSignal() {
+    const npaSignal = this.element.getAttribute('always-serve-npa');
+    if (npaSignal == undefined) {
+      return Promise.resolve(false);
+    }
+    if (npaSignal == '') {
+      return Promise.resolve(true);
+    }
+    return Services.geoForDocOrNull(this.element).then((geoService) => {
+      if (geoService) {
```

Comment:
When might this service be unavailable?

I wonder if failing closed (IE, with NPA) would be safer?

Reply:
It would be unavailable if the pub mistaken forgets to add the amp-geo component. So, I agree that it makes sense to err on the safer side here.